packages/elements/src/styles/globals.ts

```diff
@@ -177,18 +177,18 @@ export const elVariables = css`
       --color-grey-light: #f2f2f2;
 
       /** shades of blue color variables and accent colors */
-      --color-blue-light: ${colorBlueLight};
-      --color-blue-light2: ${colorBlueLight2};
-      --color-blue-dark: ${colorBlueDark};
-      --color-blue-dark2: ${colorBlueDark2};
-      --color-accent-orange: ${colorAccentOrange};
+      --color-blue-light: #23a4de;
+      --color-blue-light2: #7bc9eb;
+      --color-blue-dark: #262f69;
+      --color-blue-dark2: #31448f;
+      --color-accent-orange: #ffb71b;
```

Comment:
why did you remove the vars? Just curious and if the consts are no longer used then perhaps they should be deleted too? Thinking about SRP => no duplicated vars

Reply:
It's because I needed to be able to copy-paste the variables list into the docs when I update it and it wasn't obvious when I did this where the variables in the template literals came from - was easier to document this way, not ideal I appreciate

Reply:
ahhh I see, I'm just worried that having the hex' declared in 2 places may become confusing if there were to be changes to colours. Like if you said to me "Ashleigh, please update the danger colour" I would likely find one variable and change that variable and likely not realise there are 2 separate declarations. Would we be able to delete the TS consts to avoid confusion or too much of a hassle?